packages/@aws-cdk/integ-runner/lib/runner/runner-base.ts

```diff
@@ -157,8 +197,8 @@ export abstract class IntegRunner {
         ...options.env,
       },
     });
-    this.cdkOutDir = options.integOutDir ?? `${CDK_OUTDIR_PREFIX}.${testName}`;
-    this.cdkApp = `node ${parsed.base}`;
+    this.cdkOutDir = options.integOutDir ?? `${CDK_OUTDIR_PREFIX}.${this.testName}`;
```

Comment:
`this.testName` needs to be the `baseTestName` from `testNameFromFile`

Reply:
And actually can you just add `baseTestName` to the `TestNameParts`. Maybe call it `stableTestName`? The `testName` (really the `testDisplayName`) is something that changes depending on where you run the test from, but we also need a stable test name that _doesn't_ change to use [here](https://github.com/aws/aws-cdk/blob/777953106ac550b058fdaa3ccde25b62be07defa/packages/%40aws-cdk/integ-runner/lib/runner/integ-test-suite.ts#L159)

Reply:
Yes I'm running into some trouble with the other change I'm trying to make here as well. They seem to come together.